db/seeds.rb

```diff
@@ -7,19 +7,32 @@
+forth_user = User.create(name: 'Amanda Lee', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Dietitian from New York')
+fifth_user = User.create(name: 'Mark Brown', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Entrepreneur from London')
+sixth_user = User.create(name: 'Emily Davis', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Interior designer from Paris')
+seventh_user = User.create(name: 'David Kim', photo: 'https://unsplash.com/photos/F_-0BxGuVvo', bio: 'Communication expert from Seoul')
 
+first_post = Post.create(author: first_user, title: 'Solo', text: '10 Reasons Why Traveling Solo Can Change Your Life Forever')
+second_post = Post.create(author: second_user, title: 'Meditation', text: 'The Benefits of Meditation: How It Can Improve Your Life')
+third_post = Post.create(author: third_user, title: 'Rescue', text: 'Why You Should Consider Adopting a Rescue Dog Today')
+forth_post = Post.create(author: forth_user, title: 'Healthy', text: 'Healthy Eating Habits: Tips for a Balanced and Nutritious Die')
+fifth_post = Post.create(author: fifth_user, title: 'Online', text: 'How to Start a Successful Online Business from Scratch')
+sixth_post = Post.create(author: sixth_user, title: 'Home', text: '5 Simple Home Improvement Projects to Enhance Your Living Space')
+seventh_post = Post.create(author: seventh_user, title: 'Public', text: 'How to master the art of public speaking')
 
-first_post = Post.create(author: first_user, title: 'Hello', text: 'This is my first post')
-second_post = Post.create(author: first_user, title: 'Hello', text: 'This is my second post')
-third_post = Post.create(author: first_user, title: 'Hello', text: 'This is my third post')
-forth_post = Post.create(author: first_user, title: 'Hello', text: 'This is my forth post')
 
 
-Comment.create(post: first_post, author: second_user, text: 'Hi I am comment 1')
-Comment.create(post: first_post, author: second_user, text: 'Hi I am comment 2' )
-Comment.create(post: first_post, author: second_user, text: 'Hi I am comment 3' )
-Comment.create(post: first_post, author: second_user, text: 'Hi I am comment 4' )
-Comment.create(post: first_post, author: second_user, text: 'Hi I am comment 5' )
-Comment.create(post: first_post, author: second_user, text: 'Hi I am comment 6' )
\ No newline at end of file
+
+
+
+
+Comment.create(post: first_post, author: second_user, text: 'Hi this is comment 1' )
+Comment.create(post: first_post, author: first_user, text: 'Hi this is comment 2' )
+Comment.create(post: first_post, author: second_user, text: 'Hi this is comment 3' )
+Comment.create(post: first_post, author: forth_user, text: 'Hi this is comment 4' )
+Comment.create(post: first_post, author: fifth_user, text: 'Hi this is comment 5' )
+Comment.create(post: first_post, author: sixth_user, text: 'Hi this is comment 6' )
+Comment.create(post: first_post, author: seventh_user, text: 'Hi this is comment 7' )
```

Comment:
- [ ] I encountered an error while running `rails db:seed` caused by creating comments. I fix it using this syntax 👇🏻

```ruby 
Comment.create(post: first_post, author_id: second_user.id, text: 'Hi this is comment 1' )
Comment.create(post: first_post, author_id: first_user.id, text: 'Hi this is comment 2' )
```

- [ ] After trying different manners of creating a user using the seed and ruby console, I'm still getting an empty array of users. Kindly fix this and make sure that The seed run correctly. 💯 👍🏻 

![image](https://user-images.githubusercontent.com/79058364/231904923-ef82d410-c90a-4fa4-b2f7-59d34fa6920a.png)